src/tss2-tcti/tcti-mssim.c

```diff
@@ -284,7 +284,13 @@ tcti_mssim_get_poll_handles (
     *num_handles = 1;
     if (handles != NULL) {
 #ifdef _WIN32
-        *handles = tcti_mssim->tpm_sock;
+        HANDLE hEvent = WSACreateEvent();
+        if (WSAEventSelect(tcti_mssim->tpm_sock, hEvent, FD_READ | FD_WRITE) == 0){
+            *handles = hEvent;
+        } else {
+            WSACloseEvent(hEvent);
+            return TSS2_TCTI_RC_BAD_VALUE;
+        }
```

Comment:
Removes a block since we can return and then don't need an else clause. Also space on closing parenethesis for if statement and opening bracket.

```C
        HANDLE hEvent = WSACreateEvent();
        if (WSAEventSelect(tcti_mssim->tpm_sock, hEvent, FD_READ | FD_WRITE)) {
            WSACloseEvent(hEvent);
            return TSS2_TCTI_RC_BAD_VALUE;
        } 
        *handles = hEvent;
```

Reply:
I have made the changes to the code as requested